esp/esdllib/esdl_def.cpp

```diff
@@ -817,28 +828,28 @@ class EsdlDefStruct : public EsdlDefObject, implements IEsdlDefStruct
     {
         //DBGLOG("   Loading Element %s", method_tag.getValue("name"));
         EsdlDefElement *et = new EsdlDefElement(tag, esdl);
-        children.append(*dynamic_cast<IEsdlDefObject*>(et));
+        addChild(dynamic_cast<IEsdlDefObject*>(et));
```

Comment:
@mayx I know it was already this way, but since you are optimizing... these should all work and be faster as static_cast, can you change and test?